2020-09-24 00:01:32.142 [error] <0.376.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_6' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.684.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.710.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.142 [error] <0.372.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_2' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.689.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.706.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.142 [error] <0.377.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_7' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.687.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.705.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.143 [error] <0.371.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_1' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.685.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.702.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.143 [error] <0.373.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_3' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.681.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.701.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.143 [error] <0.375.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_5' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.686.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.703.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.143 [error] <0.378.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_8' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.682.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.709.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.143 [error] <0.370.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.683.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.708.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.143 [error] <0.379.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_9' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.688.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.704.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.143 [error] <0.374.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_4' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.690.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.707.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.159 [error] <0.371.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_1' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.685.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.702.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.161 [error] <0.377.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_7' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.687.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.705.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.161 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_1' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 1, [{name,'Elixir.Worker.Watcher.Broadway.Producer_1'},{hibernate_after,15000}]) at <0.371.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.685.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.702.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.161 [error] <0.372.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_2' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.689.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.706.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.161 [error] <0.378.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_8' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.682.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.709.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.161 [error] <0.374.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_4' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.690.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.707.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.161 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_7' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 7, [{name,'Elixir.Worker.Watcher.Broadway.Producer_7'},{hibernate_after,15000}]) at <0.377.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.687.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.705.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.162 [error] <0.375.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_5' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.686.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.703.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.162 [error] <0.373.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_3' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.681.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.701.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.162 [error] <0.370.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.683.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.708.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.162 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_2' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 2, [{name,'Elixir.Worker.Watcher.Broadway.Producer_2'},{hibernate_after,15000}]) at <0.372.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.689.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.706.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.162 [error] <0.379.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_9' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.688.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.704.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.163 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_8' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 8, [{name,'Elixir.Worker.Watcher.Broadway.Producer_8'},{hibernate_after,15000}]) at <0.378.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.682.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.709.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.163 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_4' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 4, [{name,'Elixir.Worker.Watcher.Broadway.Producer_4'},{hibernate_after,15000}]) at <0.374.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.690.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.707.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.163 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_3' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 3, [{name,'Elixir.Worker.Watcher.Broadway.Producer_3'},{hibernate_after,15000}]) at <0.373.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.681.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.701.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.164 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_5' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 5, [{name,'Elixir.Worker.Watcher.Broadway.Producer_5'},{hibernate_after,15000}]) at <0.375.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.686.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.703.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.164 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 0, [{name,'Elixir.Worker.Watcher.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.370.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.683.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.708.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.164 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_9' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 9, [{name,'Elixir.Worker.Watcher.Broadway.Producer_9'},{hibernate_after,15000}]) at <0.379.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.688.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.704.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.165 [error] <0.376.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_6' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.684.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.710.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.166 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_6' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 6, [{name,'Elixir.Worker.Watcher.Broadway.Producer_6'},{hibernate_after,15000}]) at <0.376.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.684.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.710.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.186 [error] <0.720.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_5' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.832.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.849.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.488 [error] <0.724.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_9' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.823.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.848.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.488 [error] <0.724.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_9' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.823.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.848.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.489 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_9' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 9, [{name,'Elixir.Worker.Watcher.Broadway.Producer_9'},{hibernate_after,15000}]) at <0.724.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.823.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.848.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.489 [error] <0.713.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_2' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.819.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.847.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.489 [error] <0.714.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_8' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.813.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.845.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.489 [error] <0.711.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_1' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.814.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.846.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.490 [error] <0.717.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_4' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.835.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.850.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.490 [error] <0.713.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_2' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.819.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.847.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.490 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_2' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 2, [{name,'Elixir.Worker.Watcher.Broadway.Producer_2'},{hibernate_after,15000}]) at <0.713.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.819.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.847.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.490 [error] <0.714.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_8' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.813.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.845.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.491 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_8' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 8, [{name,'Elixir.Worker.Watcher.Broadway.Producer_8'},{hibernate_after,15000}]) at <0.714.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.813.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.845.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.491 [error] <0.717.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_4' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.835.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.850.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.491 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_4' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 4, [{name,'Elixir.Worker.Watcher.Broadway.Producer_4'},{hibernate_after,15000}]) at <0.717.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.835.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.850.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.491 [error] <0.720.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_5' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.832.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.849.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:32.491 [error] <0.369.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_5' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 5, [{name,'Elixir.Worker.Watcher.Broadway.Producer_5'},{hibernate_after,15000}]) at <0.720.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.832.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.849.0>},60000]}} in context child_terminated
2020-09-24 00:01:32.492 [error] <0.722.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.842.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.852.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.001 [error] <0.5869.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.5984.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.5988.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.002 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 0, [{name,'Elixir.Worker.Watcher.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.5869.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.5984.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.5988.0>},60000]}} in context child_terminated
2020-09-24 00:01:33.014 [error] <0.5946.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_3' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6071.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6095.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.014 [error] <0.5972.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_9' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6077.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6097.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.014 [error] <0.5967.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_7' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6067.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6099.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.015 [error] <0.6005.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_6' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6083.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6098.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.015 [error] <0.5946.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_3' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6071.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6095.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.015 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_3' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 3, [{name,'Elixir.Worker.Watcher.Broadway.Producer_3'},{hibernate_after,15000}]) at <0.5946.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6071.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6095.0>},60000]}} in context child_terminated
2020-09-24 00:01:33.016 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_3' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 3, [{name,'Elixir.Worker.Watcher.Broadway.Producer_3'},{hibernate_after,15000}]) at <0.5946.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 00:01:33.016 [error] <0.5972.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_9' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6077.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6097.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.016 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_9' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 9, [{name,'Elixir.Worker.Watcher.Broadway.Producer_9'},{hibernate_after,15000}]) at <0.5972.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6077.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6097.0>},60000]}} in context shutdown_error
2020-09-24 00:01:33.016 [error] <0.6005.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_6' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6083.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6098.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.016 [error] <0.5967.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_7' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6067.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6099.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.019 [error] <0.5970.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_5' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6089.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6109.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.020 [error] <0.5996.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_2' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6084.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6103.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.020 [error] <0.5947.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_4' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6068.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6102.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.021 [error] <0.5963.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_8' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6061.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6101.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.022 [error] <0.5962.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_1' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6092.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6107.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.022 [error] <0.5996.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_2' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6084.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6103.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.023 [error] <0.5963.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_8' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6061.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6101.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.023 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_8' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 8, [{name,'Elixir.Worker.Watcher.Broadway.Producer_8'},{hibernate_after,15000}]) at <0.5963.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6061.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6101.0>},60000]}} in context shutdown_error
2020-09-24 00:01:33.023 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_7' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 7, [{name,'Elixir.Worker.Watcher.Broadway.Producer_7'},{hibernate_after,15000}]) at <0.5967.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6067.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6099.0>},60000]}} in context shutdown_error
2020-09-24 00:01:33.023 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_6' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 6, [{name,'Elixir.Worker.Watcher.Broadway.Producer_6'},{hibernate_after,15000}]) at <0.6005.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6083.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6098.0>},60000]}} in context shutdown_error
2020-09-24 00:01:33.023 [error] <0.5947.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_4' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6068.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6102.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.023 [error] <0.5962.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_1' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6092.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6107.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.024 [error] <0.5970.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_5' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6089.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6109.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.024 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_5' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 5, [{name,'Elixir.Worker.Watcher.Broadway.Producer_5'},{hibernate_after,15000}]) at <0.5970.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6089.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6109.0>},60000]}} in context shutdown_error
2020-09-24 00:01:33.024 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_4' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 4, [{name,'Elixir.Worker.Watcher.Broadway.Producer_4'},{hibernate_after,15000}]) at <0.5947.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6068.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6102.0>},60000]}} in context shutdown_error
2020-09-24 00:01:33.024 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_2' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 2, [{name,'Elixir.Worker.Watcher.Broadway.Producer_2'},{hibernate_after,15000}]) at <0.5996.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6084.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6103.0>},60000]}} in context shutdown_error
2020-09-24 00:01:33.024 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_1' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 1, [{name,'Elixir.Worker.Watcher.Broadway.Producer_1'},{hibernate_after,15000}]) at <0.5962.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6092.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6107.0>},60000]}} in context shutdown_error
2020-09-24 00:01:33.025 [error] <0.6036.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6106.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6110.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.025 [error] <0.6036.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6106.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6110.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.025 [error] <0.5474.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_0' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 0, [{name,'Elixir.Worker.Watcher.Broadway.Producer_0'},{hibernate_after,15000}]) at <0.6036.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6106.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6110.0>},60000]}} in context shutdown_error
2020-09-24 00:01:33.025 [error] <0.5473.0> Supervisor 'Elixir.Worker.Watcher.Broadway.Supervisor' had child #Ref<0.2840247403.2066743297.71331> started with 'Elixir.Supervisor':start_link([#{id => 'Elixir.Worker.Watcher.Broadway.Producer_0',shutdown => 30000,start => {'Elixir.Broadway.Topology.ProducerStage',...}},...], [{name,'Elixir.Worker.Watcher.Broadway.ProducerSupervisor'},{strategy,one_for_one},{max_restarts,...},...]) at <0.5474.0> exit with reason shutdown in context child_terminated
2020-09-24 00:01:33.056 [error] <0.6113.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_1' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6299.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6362.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.398 [error] <0.6119.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_4' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6297.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6383.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.399 [error] <0.6122.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_7' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6335.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6402.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.399 [error] <0.6112.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_0' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6339.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6396.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.399 [error] <0.6113.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_1' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6299.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6362.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.399 [error] <0.6117.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_3' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6374.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6413.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.400 [error] <0.6120.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_5' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6323.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6404.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.400 [error] <0.6111.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_1' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 1, [{name,'Elixir.Worker.Watcher.Broadway.Producer_1'},{hibernate_after,15000}]) at <0.6113.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6299.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6362.0>},60000]}} in context child_terminated
2020-09-24 00:01:33.400 [error] <0.6119.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_4' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6297.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6383.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.400 [error] <0.6116.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_2' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6363.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6419.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.400 [error] <0.6125.0> gen_server 'Elixir.Worker.Watcher.Broadway.Producer_8' terminated with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6373.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6424.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.401 [error] <0.6111.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_4' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 4, [{name,'Elixir.Worker.Watcher.Broadway.Producer_4'},{hibernate_after,15000}]) at <0.6119.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6297.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6383.0>},60000]}} in context child_terminated
2020-09-24 00:01:33.401 [error] <0.6117.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_3' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6374.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6413.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.401 [error] <0.6120.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_5' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6323.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6404.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.401 [error] <0.6112.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_0' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6339.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6396.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.401 [error] <0.6122.0> CRASH REPORT Process 'Elixir.Worker.Watcher.Broadway.Producer_7' with 0 neighbours exited with reason: {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6335.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6402.0>},60000]}} in gen_server:call/3 line 246
2020-09-24 00:01:33.402 [error] <0.6111.0> Supervisor 'Elixir.Worker.Watcher.Broadway.ProducerSupervisor' had child 'Elixir.Worker.Watcher.Broadway.Producer_3' started with 'Elixir.Broadway.Topology.ProducerStage':start_link([{broadway,[{hibernate_after,15000},{context,context_not_set},{resubscribe_interval,100},{max_seconds,...},...]},...], 3, [{name,'Elixir.Worker.Watcher.Broadway.Producer_3'},{hibernate_after,15000}]) at <0.6117.0> exit with reason {{shutdown,{server_initiated_close,404,<<"NOT_FOUND - no queue 'watcher' in vhost '/'">>}},{gen_server,call,[<0.6374.0>,{subscribe,{'basic.consume',0,<<"watcher">>,<<>>,false,false,false,false,[]},<0.6413.0>},60000]}} in context child_terminated
2020-09-24 07:55:27.372 [error] <0.381.0>@rabbit_binary_generator:table_field_to_binary:110 CRASH REPORT Process <0.381.0> with 0 neighbours crashed with reason: no function clause matching rabbit_binary_generator:table_field_to_binary({'x-delayed-type',<<"direct">>}) line 110
2020-09-24 07:55:27.372 [error] <0.378.0>@rabbit_binary_generator:table_field_to_binary:110 Supervisor {<0.378.0>,amqp_channel_sup} had child writer started with rabbit_writer:start_link(#Port<0.7>, 1, 131072, rabbit_framing_amqp_0_9_1, <0.380.0>, {<<"client 127.0.0.1:41569 -> 127.0.0.1:5672">>,1}, false, 1000000000) at <0.381.0> exit with reason no function clause matching rabbit_binary_generator:table_field_to_binary({'x-delayed-type',<<"direct">>}) line 110 in context child_terminated
2020-09-24 08:07:47.329 [error] <0.390.0> Supervisor {<0.390.0>,amqp_channel_sup_sup} had child channel_sup started with amqp_channel_sup:start_link(network, <0.389.0>, <<"client 127.0.0.1:50593 -> 127.0.0.1:5672">>) at undefined exit with reason shutdown in context shutdown_error
2020-09-24 08:33:37.578 [error] <0.593.0> gen_server <0.593.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.395.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:33:37.579 [error] <0.593.0> CRASH REPORT Process <0.593.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.395.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:33:37.579 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,5} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,5},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.3931601110.3220570113.209438>) at <0.593.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.395.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:33:37.784 [error] <0.592.0> gen_server <0.592.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.396.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:33:37.784 [error] <0.592.0> CRASH REPORT Process <0.592.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.396.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:33:37.784 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,4} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,4},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.3931601110.3220570113.209438>) at <0.592.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.396.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:33:37.986 [error] <0.590.0> gen_server <0.590.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.397.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:33:37.986 [error] <0.590.0> CRASH REPORT Process <0.590.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.397.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:33:37.986 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,2} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,2},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.3931601110.3220570113.209438>) at <0.590.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.397.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:33:38.187 [error] <0.595.0> gen_server <0.595.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.398.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:33:38.188 [error] <0.595.0> CRASH REPORT Process <0.595.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.398.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:33:38.188 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,7} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,7},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.3931601110.3220570113.209438>) at <0.595.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.398.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:33:38.188 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,7} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,7},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.3931601110.3220570113.209438>) at <0.595.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 08:33:38.188 [error] <0.586.0> Supervisor 'Elixir.ArgosWorkers.Repo' had child 'Elixir.DBConnection.ConnectionPool' started with 'Elixir.Ecto.Repo.Supervisor':start_child({'Elixir.DBConnection.ConnectionPool',start_link,[{'Elixir.Postgrex.Protocol',[{types,'Elixir.Postgrex.DefaultTypes'},...]}]}, 'Elixir.Ecto.Adapters.Postgres', #{cache => #Ref<0.3931601110.3220570113.209437>,opts => [{timeout,15000},{pool_size,10},{pool,'Elixir.DBConnection.ConnectionPool'}],...}) at <0.587.0> exit with reason killed in context child_terminated
2020-09-24 08:33:38.188 [error] <0.586.0> Supervisor 'Elixir.ArgosWorkers.Repo' had child 'Elixir.DBConnection.ConnectionPool' started with 'Elixir.Ecto.Repo.Supervisor':start_child({'Elixir.DBConnection.ConnectionPool',start_link,[{'Elixir.Postgrex.Protocol',[{types,'Elixir.Postgrex.DefaultTypes'},...]}]}, 'Elixir.Ecto.Adapters.Postgres', #{cache => #Ref<0.3931601110.3220570113.209437>,opts => [{timeout,15000},{pool_size,10},{pool,'Elixir.DBConnection.ConnectionPool'}],...}) at <0.587.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 08:33:38.189 [error] <0.366.0> Supervisor 'Elixir.ArgosWorkers.Supervisor' had child 'Elixir.ArgosWorkers.Repo' started with 'Elixir.ArgosWorkers.Repo':start_link([]) at <0.586.0> exit with reason shutdown in context child_terminated
2020-09-24 08:33:38.692 [error] <0.616.0> gen_server <0.616.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.399.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:33:38.693 [error] <0.616.0> CRASH REPORT Process <0.616.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.399.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:33:38.693 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,7} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,7},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.3931601110.3220570114.207054>) at <0.616.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.399.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:33:38.893 [error] <0.614.0> gen_server <0.614.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.400.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:33:38.894 [error] <0.614.0> CRASH REPORT Process <0.614.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.400.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:33:38.894 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,5} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,5},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.3931601110.3220570114.207054>) at <0.614.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.400.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:33:39.097 [error] <0.613.0> gen_server <0.613.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.401.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:33:39.097 [error] <0.613.0> CRASH REPORT Process <0.613.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.401.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:33:39.097 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,4} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,4},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.3931601110.3220570114.207054>) at <0.613.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.401.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:33:39.298 [error] <0.617.0> gen_server <0.617.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.402.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:33:39.298 [error] <0.617.0> CRASH REPORT Process <0.617.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.402.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:33:39.298 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,8} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,8},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.3931601110.3220570114.207054>) at <0.617.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.402.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:33:39.298 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,8} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,8},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.3931601110.3220570114.207054>) at <0.617.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 08:33:39.299 [error] <0.607.0> Supervisor 'Elixir.ArgosWorkers.Repo' had child 'Elixir.DBConnection.ConnectionPool' started with 'Elixir.Ecto.Repo.Supervisor':start_child({'Elixir.DBConnection.ConnectionPool',start_link,[{'Elixir.Postgrex.Protocol',[{types,'Elixir.Postgrex.DefaultTypes'},...]}]}, 'Elixir.Ecto.Adapters.Postgres', #{cache => #Ref<0.3931601110.3220570114.207053>,opts => [{timeout,15000},{pool_size,10},{pool,'Elixir.DBConnection.ConnectionPool'}],...}) at <0.608.0> exit with reason killed in context child_terminated
2020-09-24 08:33:39.299 [error] <0.607.0> Supervisor 'Elixir.ArgosWorkers.Repo' had child 'Elixir.DBConnection.ConnectionPool' started with 'Elixir.Ecto.Repo.Supervisor':start_child({'Elixir.DBConnection.ConnectionPool',start_link,[{'Elixir.Postgrex.Protocol',[{types,'Elixir.Postgrex.DefaultTypes'},...]}]}, 'Elixir.Ecto.Adapters.Postgres', #{cache => #Ref<0.3931601110.3220570114.207053>,opts => [{timeout,15000},{pool_size,10},{pool,'Elixir.DBConnection.ConnectionPool'}],...}) at <0.608.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 08:33:39.299 [error] <0.366.0> Supervisor 'Elixir.ArgosWorkers.Supervisor' had child 'Elixir.ArgosWorkers.Repo' started with 'Elixir.ArgosWorkers.Repo':start_link([]) at <0.607.0> exit with reason shutdown in context child_terminated
2020-09-24 08:33:39.801 [error] <0.635.0> gen_server <0.635.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.403.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:33:39.801 [error] <0.635.0> CRASH REPORT Process <0.635.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.403.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:33:39.801 [error] <0.625.0> Supervisor {<0.625.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.624.0>,10} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,10},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.624.0>, #Ref<0.3931601110.3220570121.207629>) at <0.635.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.403.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:06.091 [error] <0.598.0> gen_server <0.598.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.391.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:06.092 [error] <0.598.0> CRASH REPORT Process <0.598.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.391.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:06.092 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,10} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,10},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.911316134.267780105.46308>) at <0.598.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.391.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:06.298 [error] <0.591.0> gen_server <0.591.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.393.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:06.298 [error] <0.591.0> CRASH REPORT Process <0.591.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.393.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:06.299 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,3} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,3},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.911316134.267780105.46308>) at <0.591.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.393.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:06.501 [error] <0.596.0> gen_server <0.596.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.394.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:06.501 [error] <0.596.0> CRASH REPORT Process <0.596.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.394.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:06.501 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,8} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,8},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.911316134.267780105.46308>) at <0.596.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.394.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:06.705 [error] <0.593.0> gen_server <0.593.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.396.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:06.705 [error] <0.593.0> CRASH REPORT Process <0.593.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.396.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:06.705 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,5} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,5},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.911316134.267780105.46308>) at <0.593.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.396.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:06.705 [error] <0.588.0> Supervisor {<0.588.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.587.0>,5} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,5},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.587.0>, #Ref<0.911316134.267780105.46308>) at <0.593.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 08:36:06.705 [error] <0.585.0> Supervisor 'Elixir.ArgosWorkers.Repo' had child 'Elixir.DBConnection.ConnectionPool' started with 'Elixir.Ecto.Repo.Supervisor':start_child({'Elixir.DBConnection.ConnectionPool',start_link,[{'Elixir.Postgrex.Protocol',[{types,'Elixir.Postgrex.DefaultTypes'},...]}]}, 'Elixir.Ecto.Adapters.Postgres', #{cache => #Ref<0.911316134.267780105.46307>,opts => [{timeout,15000},{pool_size,10},{pool,'Elixir.DBConnection.ConnectionPool'}],...}) at <0.587.0> exit with reason killed in context child_terminated
2020-09-24 08:36:06.706 [error] <0.585.0> Supervisor 'Elixir.ArgosWorkers.Repo' had child 'Elixir.DBConnection.ConnectionPool' started with 'Elixir.Ecto.Repo.Supervisor':start_child({'Elixir.DBConnection.ConnectionPool',start_link,[{'Elixir.Postgrex.Protocol',[{types,'Elixir.Postgrex.DefaultTypes'},...]}]}, 'Elixir.Ecto.Adapters.Postgres', #{cache => #Ref<0.911316134.267780105.46307>,opts => [{timeout,15000},{pool_size,10},{pool,'Elixir.DBConnection.ConnectionPool'}],...}) at <0.587.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 08:36:06.706 [error] <0.366.0> Supervisor 'Elixir.ArgosWorkers.Supervisor' had child 'Elixir.ArgosWorkers.Repo' started with 'Elixir.ArgosWorkers.Repo':start_link([]) at <0.585.0> exit with reason shutdown in context child_terminated
2020-09-24 08:36:07.207 [error] <0.612.0> gen_server <0.612.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.397.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:07.207 [error] <0.612.0> CRASH REPORT Process <0.612.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.397.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:07.208 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,3} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,3},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.911316134.267780097.48746>) at <0.612.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.397.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:07.408 [error] <0.618.0> gen_server <0.618.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.399.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:07.408 [error] <0.618.0> CRASH REPORT Process <0.618.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.399.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:07.409 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,9} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,9},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.911316134.267780097.48746>) at <0.618.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.399.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:07.611 [error] <0.619.0> gen_server <0.619.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.401.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:07.611 [error] <0.619.0> CRASH REPORT Process <0.619.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.401.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:07.612 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,10} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,10},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.911316134.267780097.48746>) at <0.619.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.401.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:40.469 [error] <0.389.0> gen_server <0.389.0> terminated with reason: socket_closed_unexpectedly
2020-09-24 08:36:40.469 [error] <0.389.0> CRASH REPORT Process <0.389.0> with 0 neighbours exited with reason: socket_closed_unexpectedly in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:40.469 [error] <0.387.0> Supervisor {<0.387.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.388.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.389.0> exit with reason socket_closed_unexpectedly in context child_terminated
2020-09-24 08:36:40.469 [error] <0.387.0> Supervisor {<0.387.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.388.0>, {amqp_params_network,<<"guest">>,{plaintext,<<"guest">>},<<"/">>,"localhost",5672,0,0,10,50000,none,...}) at <0.389.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 08:36:40.682 [error] <0.621.0> gen_server <0.621.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.403.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:40.683 [error] <0.621.0> CRASH REPORT Process <0.621.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.403.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:40.683 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,9} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,9},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.911316134.267780097.48746>) at <0.621.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.403.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:40.885 [error] <0.615.0> gen_server <0.615.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.404.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:40.885 [error] <0.615.0> CRASH REPORT Process <0.615.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.404.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:40.885 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,6} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,6},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.911316134.267780097.48746>) at <0.615.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.404.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:41.086 [error] <0.620.0> gen_server <0.620.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.405.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:41.086 [error] <0.620.0> CRASH REPORT Process <0.620.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.405.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:41.086 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,3} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,3},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.911316134.267780097.48746>) at <0.620.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.405.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:41.289 [error] <0.616.0> gen_server <0.616.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.406.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168
2020-09-24 08:36:41.289 [error] <0.616.0> CRASH REPORT Process <0.616.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.406.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in gen_server:handle_common_reply/8 line 796
2020-09-24 08:36:41.289 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,7} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,7},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.911316134.267780097.48746>) at <0.616.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.DBConnection.ConnectionError',message => <<"client #PID<0.406.0> stopped: ** (UndefinedFunctionError) function Jason.decode!/1 is undefined (module Jason is not available)\n    Jason.decode!(\"[{\\\"id\\\": 4}, {\\\"id\\\": 5}]\")\n    (postgrex 0.15.6) lib/postgrex/type_module.ex:897: Postgrex.DefaultTypes.\"Elixir.Postgrex.Extensions.JSONB\"/6\n    (postgrex 0.15.6) lib/postgrex/protocol.ex:2870: Postgrex.Protocol.rows_recv/4\n    (postgrex 0.15.6) lib/post...">>,...} in 'Elixir.DBConnection.Connection':handle_cast/2 line 168 in context child_terminated
2020-09-24 08:36:41.290 [error] <0.609.0> Supervisor {<0.609.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.608.0>,7} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,7},{types,'Elixir.Postgrex.DefaultTypes'},{port,5432},{repo,'Elixir.ArgosWorkers.Repo'},...], <0.608.0>, #Ref<0.911316134.267780097.48746>) at <0.616.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 08:36:41.290 [error] <0.607.0> Supervisor 'Elixir.ArgosWorkers.Repo' had child 'Elixir.DBConnection.ConnectionPool' started with 'Elixir.Ecto.Repo.Supervisor':start_child({'Elixir.DBConnection.ConnectionPool',start_link,[{'Elixir.Postgrex.Protocol',[{types,'Elixir.Postgrex.DefaultTypes'},...]}]}, 'Elixir.Ecto.Adapters.Postgres', #{cache => #Ref<0.911316134.267780097.48745>,opts => [{timeout,15000},{pool_size,10},{pool,'Elixir.DBConnection.ConnectionPool'}],...}) at <0.608.0> exit with reason killed in context child_terminated
2020-09-24 08:36:41.290 [error] <0.607.0> Supervisor 'Elixir.ArgosWorkers.Repo' had child 'Elixir.DBConnection.ConnectionPool' started with 'Elixir.Ecto.Repo.Supervisor':start_child({'Elixir.DBConnection.ConnectionPool',start_link,[{'Elixir.Postgrex.Protocol',[{types,'Elixir.Postgrex.DefaultTypes'},...]}]}, 'Elixir.Ecto.Adapters.Postgres', #{cache => #Ref<0.911316134.267780097.48745>,opts => [{timeout,15000},{pool_size,10},{pool,'Elixir.DBConnection.ConnectionPool'}],...}) at <0.608.0> exit with reason reached_max_restart_intensity in context shutdown
2020-09-24 08:36:41.290 [error] <0.366.0> Supervisor 'Elixir.ArgosWorkers.Supervisor' had child 'Elixir.ArgosWorkers.Repo' started with 'Elixir.ArgosWorkers.Repo':start_link([]) at <0.607.0> exit with reason shutdown in context child_terminated
